packages/lavapack/.depcheckrc

```diff
@@ -0,0 +1,3 @@
+ignores:
+  # monorepo deps
+  - "ava"
```

Comment:
because `depcheck` is _not_ monorepo-aware, `ava` will be referenced in e.g., `test/index.js`, but it will not appear in the `devDependencies` of `lavapack/package.json`.

any dev dep in a workspace which is required from code (unsure if this applies to dev deps only referenced in e.g., `package.json`'s `scripts` prop) but is only in the workspace root's dev deps would need to be in this `ignore` property.